docs/docs/unit-testing.md

```diff
@@ -27,31 +27,26 @@ npm install --save-dev jest babel-jest react-test-renderer identity-obj-proxy 'b
 
 Because Gatsby handles its own Babel configuration, you will need to manually
 tell Jest to use `babel-jest`. The easiest way to do this is to add a `"jest"`
-section in your `package.json`. You can set up some useful defaults at the same
+section in your `jest.config.js`. You can set up some useful defaults at the same
```

Comment:
`jest.config.js` should directly export the defaults, right?

There shouldn't be a `"jest"` section in `jest.config.js`.

Reply:
Fixed! Thanks a lot!!